src/components/LiquidityMining/Create/Steps/PairAndReward/AssetSelector/index.tsx

```diff
@@ -1,77 +1,249 @@
 
   useEffect(() => {
     if (currency0 && currency1) {
+      setTokenName('LP PAIR')
       setAssetTitle(`${unwrappedToken(currency0)?.symbol}/${unwrappedToken(currency1)?.symbol}`)
     } else if (currency0) {
+      setTokenName(unwrappedToken(currency0)?.name)
       setAssetTitle(unwrappedToken(currency0)?.symbol || null)
+    } else {
+      setTokenName(undefined)
+      setAssetTitle(`SELECT ${campaingType === CampaignType.TOKEN ? 'TOKEN' : 'PAIR'}`)
     }
-  }, [currency0, currency1])
+  }, [currency0, currency1, campaingType])
+
+  const handleDismiss = () => {
+    setAssetTitle(null)
+    if (onResetCurrency) onResetCurrency()
+    setTokenName(undefined)
+  }
 
   return (
-    <Flex flexDirection="column">
-      <Box mb="16px">
-        <TYPE.small fontWeight="600" color="text4" letterSpacing="0.08em">
-          {title}
-        </TYPE.small>
-      </Box>
-      <Box>
-        <Card selectable onClick={onClick}>
-          <Flex height="26px" width="100%" justifyContent="center" alignItems="center">
-            <Box mr={currency0 ? '8px' : '0'}>
-              <AssetLogo currency0={currency0} currency1={currency1} />
-            </Box>
-            <Box>
-              <TYPE.body
-                lineHeight="20px"
-                color="white"
-                data-testid={title.toLocaleLowerCase().replace(' ', '-') + '-select'}
+    <Flex flexDirection={'column'} key={index}>
+      <SmoothGradientCard
+        selectable
+        isToken={true}
+        active={currency0 !== undefined}
+        paddingBottom={
+          isReward && index === 0 && currency0 === undefined ? '45px' : isReward ? '30px' : '28px' + '!important'
+        }
+        width={'162px'}
+        flexDirection={'column-reverse'}
+        height={isReward ? '192px' : '150px'}
+        onClick={onClick}
+      >
+        {isReward && currency0 && (
+          <RelativeDismiss
+            onClick={event => {
+              event.stopPropagation()
+              if (isReward) handleDismiss()
+            }}
+          />
+        )}
+
+        <Flex width="100%" justifyContent="center" alignSelf="end">
+          <AssetLogo campaingType={campaingType} currency0={currency0} currency1={currency1} />
+          <Flex flexDirection={'column'}>
+            {isReward && currency0 && handleUserInput ? (
+              <RelativeContainer>
+                <StyledNumericalInput
+                  style={{
+                    width: `${
+                      rawAmount && rawAmount.length > 16 ? '150' : rawAmount ? 17 + rawAmount?.length * 8 : 25
+                    }px`,
+                  }}
+                  onClick={event => event.stopPropagation()}
+                  placeholder="0"
+                  value={rawAmount ? rawAmount : ''}
+                  onUserInput={handleUserInput}
+                />
+              </RelativeContainer>
+            ) : (
+              //deleted data-testid={title.toLocaleLowerCase().replace(' ', '-') + '-select'}
```

Comment:
We shouldn't leave this comment here. We should check if data-testid is needed and delete the comment

Reply:
well it needs to be refactored since it used to be one reward and now we have mutliple which are conditonally rendered will make issue for it...

Reply:
I made an issue  #1041